components/treeview/index.css

```diff
@@ -97,7 +97,7 @@ governing permissions and limitations under the License.
 
   transition: transform ease var(--spectrum-global-animation-duration-100);
 
-  pointer-events: all !important;
+  pointer-events: all;
```

Comment:
@tacoelho Still seeing some commits related to the removal of `!important` leaking in here, can you remove them from the history of this branch? You should be able to do so with `git rebase -i 85906111f582c037cd1b332e954601371c2a8cb8~1`, then delete the lines that reference commits which don't modify READMEs.

Reply:
Rebased.